electricitymap/contrib/parsers/tests/test_CZ.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime
+from importlib import resources
+
+import pytest
+from requests_mock import POST
+
+from electricitymap.contrib.lib.types import ZoneKey
+from electricitymap.contrib.parsers import CZ
+from electricitymap.contrib.parsers.lib.exceptions import ParserException
+
+
+@pytest.fixture
+def mock_good_response(adapter):
+    """
+    Mocks the CEPS API with a "good" response containing expected data.
+    """
+    mock_xml_content = (
+        resources.files("electricitymap.contrib.parsers.tests.mocks.CZ")
+        .joinpath("CrossborderPowerFlows.xml")
+        .read_text()
+    )
+
+    adapter.register_uri(
+        POST,
+        CZ.url,
+        text=mock_xml_content
+    )
+
+@pytest.fixture
+def mock_no_data_response(adapter):
+    """
+    Mocks the CEPS API with a "bad" response that is missing the <data> tag.
+    (This fixture is requested by the exception test)
+    """
+
+    mock_xml_content = (
+        '<root xmlns="https://www.ceps.cz/CepsData/StructuredData/1.0">'
+        '<information><name>Cross-border power flows</name></information>'
+        '<series><serie id="value1" name="PSE Actual [MW]" /></series>'
+        '</root>'
+    )
+
+    adapter.register_uri(
+        POST,
+        CZ.url,
+        text=mock_xml_content
+    )
+
+# Using a fixed date that exists in the mock XML data
+TARGET_DATETIME = datetime(2025, 10, 31, 18, 0, 0)
+
+
+# CT1 e CT4
```

Comment:
This comment is pretty much meaningless and don't really say anything about the test.